### DISCUSSION TITLE
List of Azure AD protected resources?

### DISCUSSION THREAD

**Discussion by schwichti** [Azure/azure-workload-identity#861] [Q&A]
Hi,

the introduction says "Workloads deployed in Kubernetes clusters require Azure AD application credentials or managed identities to access Azure AD protected resources, such as Azure Key Vault and Microsoft Graph."
Is there a list of Azure AD protected resources? If yes, tutorials for each would be appreciated.

**Comment by aramase** [comment]
Hello 👋🏻 

> Is there a list of Azure AD protected resources? If yes, tutorials for each would be appreciated.

Thanks for the feedback. Any Azure resource that supports managed identity for access will work with Workload Identity Federation (WIF). Given there are plenty of azure resources that support RBAC with managed identity, I don't think it's feasible to add a tutorial for every one of them in this project. The example for getting a token defined in this project would work similarly for any other azure resource you're trying to access.
